plugins/jobs/server/job_rest.py

```diff
@@ -48,54 +57,26 @@ def listJobs(self, params):
             user = self.model('user').load(
                 params['userId'], user=currentUser, level=AccessType.READ)
 
-        jobs = self.model('job', 'jobs').list(
+        return list(self.model('job', 'jobs').list(
```

Comment:
@mgrauer I'm not sure whether you mean a list comprehension or a list literal. 
I don't see how either is really appropriate here, the constructor seems correct to me.